wiz-admission-controller/values.yaml

```diff
@@ -0,0 +1,92 @@
+# Default values for wiz-admission-controller.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+wizApiToken:
+  clientId: ""
+  clientToken: ""
+  env: "prod" # (prod/ gov)
```

Comment:
IMO `prod` should be an internal term and not exposed to clients. maybe `endpoint` or `prefix` (`=app`) should be a better fit here.